### ISSUE TITLE
`terminal.brightBlack` for the dark themes is unusable

### ISSUE THREAD

**Issue by PurpleMyst** [ayu-theme/ayu-colors#32]
`themes/dark.yaml` and `themes/mirage.yaml` give `#0A0000` for both `black` and `brightBlack`. That's fine for the former since it's mostly used as a background, but the latter gives a horrid contrast ratio for any tools which use bright black on black: 1.09:1 against the dark background, 1.28:1 on mirage, so anything rendering dim text with `90m` is basically invisible.

Using a gray from the palette instead, e.g. `brightBlack: $palette.gray.l4` (`#5A6673`, 3.25:1 on dark, 2.76:1 on mirage) or `gray.l5` (`#667381`, 3.93:1 / 3.34:1), might be better? Both are already in the theme, mirage's `white`/`brightWhite` use them, and `gray.l4` is the same gray as `syntax.comment`.
